### ISSUE TITLE
Notifications not sending

### ISSUE THREAD

**Issue by frndlyneighborhoodspiderman** [godstale/retrowatch#7]
The app will connect to my watch and the time will display but no notifications are going through

**Comment by srijansaxena11** [comment]
same prob here. please help.

**Comment by dhrmik** [comment]
Yes , please provide the complete code for time as well as notification showing on screen. Please help at earliest

**Comment by dhrmik** [comment]
No call notifications are showing please provide the code.

**Comment by srijansaxena11** [comment]
if anyone gets any help regarding this or make it work kindly post it here

**Comment by srijansaxena11** [comment]
no replies yet. :( anyone any hint, idea or guess at least?

**Comment by dhrmik** [comment]
I have made it working .

**Comment by frndlyneighborhoodspiderman** [comment]
@dhrmik could you please share the code?

**Comment by dhrmik** [comment]
Make this changes in code

1. Replace all mySerial with Serial

Change 5,6 pin of Arduino with rx , tx. Rx of Bluetooth to tx of Arduino and tx of Bluetooth to rx of Arduino.

Change baud rate to 19600

Remove line of connection in start which ..........(5,6) //rx,tx

**Comment by srijansaxena11** [comment]
@dhrmik can you send the code please

**Comment by srijansaxena11** [comment]
@frndlyneighborhoodspiderman did you make it work too?

**Comment by frndlyneighborhoodspiderman** [comment]
No, sadly

**Comment by dhrmik** [comment]
Do the changes which i had above. It will work. It worked for me.
You can contact me dharmik1211@gmail.com

**Comment by Phoenix3101** [comment]
> Do the changes which i had above. It will work. It worked for me.
> You can contact me [dharmik1211@gmail.com](mailto:dharmik1211@gmail.com)

Bro is it the procedure for u8glib_no button?

**Comment by Upstel** [comment]
try rewiring the Bluetooth module to
SCL - A5
SDA - A4
VCC - 5V
GND - GND
